tests/bsim/bluetooth/mesh/src/dfu_blob_common.c

```diff
@@ -52,7 +52,7 @@ void common_sar_conf(uint16_t addr)
 	 */
 	struct bt_mesh_sar_tx tx_set = {
 		.seg_int_step = 1,
-		.unicast_retrans_count = 3,
+		.unicast_retrans_count = 0,
```

Comment:
This will make unicast SAR pretty unstable for multi segment messages.

Reply:
In the real life - yes, but not in bsim environment.
bsim environment is deterministic and without looses. It means 100% sent frames are received (if frame collision didn't happen).
Even more, if many repeats are configured for SAR it will make the test scenario more difficult. In scenario where a couple of devices participate and send something at the same time they create a lot of collisions especially for transport acks. Transport acks collide with retransmission very often.

To solve these collisions, there is another SAR parameter `unicast_retrans_without_prog_count` that is responsible for retransmission if the segment wasn't acked. This parameter is set to 2 for DFU and BLOB tests. It means collisions will be managed to some extent on transport layer.

For bsim tests the test scenarios require to not be overloaded with extra traffic that might help in real life. This is kind of tradeoff.